R-Interface/jasprcpp.h

```diff
@@ -40,8 +40,8 @@ SEXP jaspRCPP_allColumnNamesDataset(SEXP encoded);
 SEXP jaspRCPP_RunSeparateR(SEXP code);
 
 				//Custom parseEvals to make sure sink is set (to capture output)
-void			jaspRCPP_parseEvalQNT(const std::string & code,	bool preface = true);
-RInside::Proxy	jaspRCPP_parseEval(const std::string & code,	bool preface = true);
+void			jaspRCPP_parseEvalQNT(const std::string & code,	bool setWd = false,	bool preface = true);
+RInside::Proxy	jaspRCPP_parseEval(const std::string & code,	bool setWd = true,	bool preface = true);
```

Comment:
I would make the defaults the same for both.

Reply:
OK, done

Reply:
Seems like it would have been less work to change the default the other way round, haha.

So, I'll leave it to you. You can either merge like this, or switch the default to `false`